### PR TITLE
Update config docs

### PR DESCRIPTION
**Pull request by tyrannosaurus-becks**
This PR updates the docs to match the changes in #180.

### CODE REVIEW

File: docs/docs/configuration.mdx [hunk 1]
```diff
@@ -2,31 +2,81 @@
...
+| --log-level        | APPROZIUM_LOG_LEVEL        | info       | Supported selections are "trace", "debug", "info", "warn", "error", "fatal", and "panic". Upper case may be used.                                                                                                                                                                                                                                                         |
+| --log-format       | APPROZIUM_LOG_FORMAT       | text       | Also supports "json".                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (timothydnguyen) [inline docs/docs/configuration.mdx:27]: Can you please include `"text"` as a supported selection as well to match the above desc?

File: docs/docs/configuration.mdx [hunk 2]
```diff
@@ -2,31 +2,81 @@
...
+| --vault-token-path | APPROZIUM_VAULT_TOKEN_PATH |            | Optional, if set it will cause the latest Vault token to always be pulled from the given file. This option takes precedence over --vaulttoken                                                                                                                                                                                                                             |
+| --vault-token      | APPROZIUM_VAULT_TOKEN      |            | Optional, if set it will be used at the Vault token.                                                                                                                                                                                                                                                                                                                      |
```
Comment (timothydnguyen) [inline docs/docs/configuration.mdx:44]: typo in description? ("at" -> "as")

File: docs/docs/configuration.mdx [hunk 3]
```diff
@@ -2,31 +2,81 @@
...
+| --vault-addr       | APPROZIUM_VAULT_ADDR       |            | Required if using Vault as the secrets manaager.                                                                                                                                                                                                                                                                                                                          |
```
Comment (timothydnguyen) [inline docs/docs/configuration.mdx:45]: typo in description

File: docs/docs/configuration.mdx [hunk 4]
```diff
@@ -2,31 +2,81 @@
...
+|--------------------|----------------------------|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --config           | APPROZIUM_CONFIG_FILE_PATH | false      | Optional, set it to path containing a YAML config file called `approzium.config.yml`                                                                                                                                                                                                                             |
```
Comment (timothydnguyen) [inline docs/docs/configuration.mdx:52]: config file no longer has to be named `approzium.config.yml` with #180, right?

File: docs/docs/configuration.mdx (hunk 1 above)
Comment (tyrannosaurus-becks) [reply docs/docs/configuration.mdx]: Yep! https://github.com/cyralinc/approzium/pull/181/commits/bd9c01350393297b581861a0c9651774617bd329

File: docs/docs/configuration.mdx (hunk 2 above)
Comment (tyrannosaurus-becks) [reply docs/docs/configuration.mdx]: Nice catch! You have eyes like a....

laser eyes

Comment (tyrannosaurus-becks) [reply docs/docs/configuration.mdx]: https://github.com/cyralinc/approzium/pull/181/commits/9c9f6dbcf83d42d066c6f3dbae1763ff6d2b73a2

File: docs/docs/configuration.mdx (hunk 3 above)
Comment (tyrannosaurus-becks) [reply docs/docs/configuration.mdx]: https://github.com/cyralinc/approzium/pull/181/commits/2fca5a7d41af828adae7ab530a252fb503acdc5d

Typos are to you what cucumbers are to this cat!

cat laser eyes

File: docs/docs/configuration.mdx (hunk 4 above)
Comment (tyrannosaurus-becks) [reply docs/docs/configuration.mdx]: Good call! https://github.com/cyralinc/approzium/pull/181/commits/9e00ab04db20c010345d50a4df1402754c8a662f

**Pull request merged**